fpga_interchange/populate_chip_info.py

```diff
@@ -1867,7 +1867,7 @@ def populate_chip_info(device, constids, device_config):
             device.device_resource_capnp.tileTypeList):
         flattened_tile_type = FlattenedTileType(
             device, tile_type_index, tile_type, cell_bel_mapper, constraints,
-            lut_elements, disabled_routethrus, disabled_site_pips)
+            lut_elements, disabled_routethrus)
```

Comment:
I believe this is a rebase error, as `disabled_site_pips` was recently added.

Reply:
I'll fix this shortly